pkg/webkitgtk2

```diff
@@ -0,0 +1,85 @@
+[mirrors]
+https://webkitgtk.org/releases/webkitgtk-2.20.3.tar.xz
+
+[vars]
+filesize=16623456
+sha512=69a7b27e294de6fd43240ce1d314cd1ccdbf29674c97967674459fd02720b31fa53e5412efbd93dc1e805d38e38feb7fe9a12cdd9c472471676b8db3c11734de
+pkgver=1
+
+[deps]
+cairo
+enchant
+gtk+3
```

Comment:
webkitgtk**2** requires gtk+**3** ?

Reply:
yup, it's actually webkit2gtk4 that gets installed, but often referred to as webkitgkt2. just as webkitgtk1 requires gtk+2. And webkit2gtk4 also needs gtk+2 as well for nsplugin stuff too - which I skipped.